docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,393 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
 ##### `random`
 This strategy does not change terminator weights. It does simple random selection across all terminators of the highest precedence. 
 
+## Service Hosting
```

Comment:
After coming back to this, I think these would probably all be good subtopics on 'services' too.

Reply:
As I review, I think there should be a paragraph explaining what even hosting means here before introducing all the rest.